packages/cardhost/app/services/library.ts

```diff
@@ -16,6 +20,7 @@ const size = 100;
 
 export default class LibraryService extends Service {
   @service data!: DataService;
+  @service cardLocalStorage: any;
```

Comment:
you should really use TS for your service and then use that as the type definition

Reply:
It was tough to turn CardLocalStorage into a ts file because the polyfill had no types. I removed the polyfill altogether, which means that only browsers with local storage can use this feature. However, I think that's ok given https://caniuse.com/#feat=mdn-api_window_localstorage